.github/PULL_REQUEST_TEMPLATE/default.md

```diff
@@ -0,0 +1,40 @@
+---
+labels: Needs Review
+name: Pull Request
+---
+<!--- Provide a general summary of your changes in the Title above -->
+
+## Description
+<!--- Describe your changes in detail -->
+
+## Motivation and Context
+<!--- Why is this change required? What problem does it solve? -->
+<!--- If it fixes an open issue, please link to the issue here. -->
+
+## How Has This Been Tested?
```

Comment:
I link this phrasing vs our current "how should this be tested?" as it might help reduce testing blind spots where the reviewer would be a little more critical of the steps rather than just following the instructions verbatim.

Reply:
What I like about "how should this be tested" is that it implies some instruction on how to set up the environment. I'm concerned that "how has this been tested" responses will be shorter and skip over the set-up steps.